client/payment-details/summary/test/__snapshots__/index.tsx.snap

```diff
@@ -24,9 +24,9 @@ exports[`PaymentDetailsSummary correctly renders a charge 1`] = `
               usd
             </span>
             <span
-              class="chip chip-light"
+              class="chip chip-primary"
             >
-              Paid
+              Payment authorized
```

Comment:
I was not expecting to see this change of status in the payment TBH. I guess it comes from changes in `develop` branch 🤔

Reply:
I think the snapshot is changing since we have replaced the static date showing in the capture notice, with a relative time. I hope my understanding is correct here?

Reply:
Hmm, I'd say that the date is not relevant for the status of the payment, but would need to dig into that to be sure. Unfortunately, I don't have the capacity at the moment.

Reply:
The status here seems to be correct as expected - `Payment authorized`. Not sure why it was showing `Paid` in the status earlier though. 🤔